js/projects.js

```diff
@@ -113,6 +108,11 @@
                     ]
                 };
 
+                var ecd_cft = projectInfos.find(el => el.id === "ecd.cft")
```

Comment:
I understand this works but It would be best to stay consistent with the file and use semicolons at the end of each line.

Reply:
Added.